lib/multisigPayload.example.json

```diff
@@ -0,0 +1 @@
+{"destinations": ["0xFC88f1406af22D522A74E922E8AaB170D723a665"],"values": [20],"payloads": ["0x7065cb480000000000000000000000006ecbe1db9ef729cbe972c83fb886247691fb6beb"]}
```

Comment:
This is a temp file with an example payload to add an owner to the multiSig contract